lib/CRMS/RightsPredictor.pm

```diff
@@ -204,30 +207,34 @@ sub predict_last_source_copyright {
   return if $prediction->{error};
   my $country = $self->{record}->country;
   my $predictor = $TERM_PREDICTORS->{$country};
-  my $term = $predictor->($prediction->{death_or_pub_date}, $prediction->{is_crown});
+  my $term = $predictor->(
+    $prediction->{death_or_pub_date},
+    $prediction->{is_pub},
+    $prediction->{is_crown}
+  );
   $prediction->{last_source_copyright_year} = $prediction->{death_or_pub_date} + $term;
   push @{$prediction->{desc}},
     "Last $country © $prediction->{last_source_copyright_year} ($prediction->{death_or_pub_date} + $term-year term)";
 }
 
 sub australia_term {
-  my ($death_or_pub_date, $is_crown) = @_;
+  my ($death_or_pub_date, $is_pub, $is_crown) = @_;
```

Comment:
Var `$is_pub` not used in this block. I would:

Either: indicate that with an initial underscore in the variable name: `$_is_pub` but that usage may be overloaded and is harder to read with the super-leading dollar sign.

Or: assign the value to undef if you know you aren't using it:
`my ($death_or_pub_date, undef, $is_crown) = @_;`

Reply:
I didn't know you could do that kind of assignment to `undef`. Cool, that's the approach that appeals to me.